server/services/DEPRECATEnotificationService.js

```diff
@@ -3,72 +3,55 @@ const knex = require('knex')
 const db = require('../database')
 
 const DEPRECATEnotificationService = {
-  // this is only used to count the number of notifications in calendar.ejs, and we have two other methods in this class that do the same thing.
+  // this is only used to count the number of notifications in calendar.ejs
   async getShiftNotifications(quantumId) {
     return db
-      .select(
-        knex.raw(
-          `"DESCRIPTION" as "Description", "LAST_MODIFIED_DATE_TIME" as "LastModifiedDateTime", "PROCESSED" as "Processed"`,
-        ),
-      )
-      .from('SHIFT_NOTIFICATION')
-      .where('QUANTUM_ID', '=', quantumId.toLowerCase())
+      .select('quantum_id', 'last_modified')
+      .from('shift_notification')
+      .where('quantum_id', '=', quantumId.toLowerCase())
       .union(
         db
-          .select(
-            knex.raw(
-              `"DESCRIPTION" as "Description", "LAST_MODIFIED_DATE_TIME" as "LastModifiedDateTime", "PROCESSED" as "Processed"`,
-            ),
-          )
-          .from('SHIFT_TASK_NOTIFICATION')
-          .where('QUANTUM_ID', '=', quantumId.toLowerCase()),
+          .select('quantum_id', 'last_modified')
+          .from('shift_task_notification')
+          .where('quantum_id', '=', quantumId.toLowerCase()),
       )
-      .orderBy('LastModifiedDateTime', 'desc')
       .catch((err) => {
         throw err
       })
   },
 
   getShiftNotificationsPaged(quantumId, offset, perPage) {
     return db
-      .select(
-        knex.raw(
-          `"DESCRIPTION" as "Description", "LAST_MODIFIED_DATE_TIME" as "LastModifiedDateTime", "PROCESSED" as "Processed"`,
-        ),
-      )
-      .from('SHIFT_NOTIFICATION')
-      .where('QUANTUM_ID', '=', quantumId.toLowerCase())
+      .select(knex.raw(`description AS "Description", last_modified AS "LastModifiedDateTime"`))
```

Comment:
Is there a reason to leave "DateTime" at the end of `LastModifiedDateTime`? I noticed you had removed this elsewhere in the code base. Seems like an odd choice of naming since datetime is not a type in postgresql

Reply:
I didn't want to change the template, otherwise I'd do them all and get rid of the 'as' part completely.
I thought I'd let you do it as part if your moving to rest work.

Reply:
Yeah okay, the `knex.raw` isn't needed here anyway. You can pass it an object so that we always have the same mapping when accessing the db:

```
const shiftNotificationMapping = {
  descripton: 'Description',
  "last_modified": 'LastModifiedDateTime
}

const data = await db('SHIFT_NOTIFICATION')
  .select(shiftNotificationMapping)
```
It works as a nice way to decouple the table names from our logic. Doesn't matter I guess as we're dropping access to the db from this app over the next few weeks.